Library/Homebrew/style.rb

```diff
@@ -164,7 +164,9 @@ def run_rubocop(files, output_type,
     end
 
     def run_shellcheck(files, output_type)
-      shellcheck   = Formula["shellcheck"].opt_bin/"shellcheck" if Formula["shellcheck"].any_version_installed?
+      if Formula["shellcheck"].any_version_installed? && !ENV["GITHUB_ACTIONS"]
```

Comment:
Is this version failing for some reason? Might be helpful to have a comment explaining what's going on here.

Reply:
when `shellcheck` is `Formula["shellcheck"].opt_bin/"shellcheck"`(`/home/linuxbrew/.linuxbrew/opt/shellcheck/bin/shellcheck`)
shellcheck result is success.
But when it is `which("shellcheck")`(`/usr/bin/shellcheck>`), the result is a failure, and error messages are displayed.

Reply:
@hyuraku Interesting. Is it still running but it's just an invalid version? Perhaps we could switch this to `.latest_version_installed?` and see if that works without the `GITHUB_ACTIONS`?

Reply:
I add the code (https://github.com/Homebrew/brew/pull/11939/commits/af0cd0cb6dd8db680e980442aa25b9e0298bac8d) for debug and confirm that the newest version `shellcheck` is installed by the below message from https://github.com/Homebrew/brew/pull/11939/checks?check_run_id=3485140561

```
Run brew style --display-cop-names

brew style --display-cop-names
shell: /usr/bin/bash -e {0}

env:
  HOMEBREW_DEVELOPER: 1
  HOMEBREW_NO_AUTO_UPDATE: 1

1100 files inspected, no offenses detected
"Formula[shellcheck].latest_version_installed?: true"
```

Reply:
@hyuraku so the latest version is installed but it still does not work as expected? 

Thanks!

Reply:
> so the latest version is installed but it still does not work as expected?

Yes, I agree with you.

Reply:
@hyuraku And is it definitely never displaying any errors here? Do you think it is broken, not being executed properly or outputting in a different format?

Reply:
> However, when the version is 0.7.2 (latest), no error is displayed.

I think it's fine to ignore this then. We should consider making the latest version always be required I think to avoid confusion like this in future. Can you adjust this PR to do that?

Reply:
> Can you adjust this PR to do that?

OK, I add the condition to check whether the `shellcheck` is `latest_version_installed` or not.